dm/dm.go

```diff
@@ -96,18 +111,39 @@ func main() {
 		usage()
 	}
 
+	command := args[0]
 	if *stdin {
-		fmt.Printf("reading from stdin is not yet implemented")
-		os.Exit(0)
+		if len(args) != 2 {
+			usage()
+		}
+
+		cmd := exec.Command("tar", "xf", "-")
```

Comment:
Instead of the added complexity of forking a new process, why not use a library so we don't have to make the code more complex for error handling?
https://golang.org/pkg/archive/tar/